tasks/src/main/java/com/example/task/CreateQueue.java

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.example.task;
+
+// [START cloud_tasks_create_queue]
+import com.google.cloud.tasks.v2.CloudTasksClient;
+import com.google.cloud.tasks.v2.LocationName;
+import com.google.cloud.tasks.v2.Queue;
+import com.google.cloud.tasks.v2.QueueName;
+
+public class CreateQueue {
+  /**
+   * Create a queue using the Cloud Tasks client.
+   *
+   * @param projectId the Id of the project.
+   * @param queueId the name of your Queue.
+   * @param locationId the GCP region of your queue.
```

Comment:
Typically, we have a main method that sets these values that users can modify to run this on their own.  Take a look at both our style guide and other smaples

Reply:
Added!